spec/features/shelterreviews/shelter_review_creation_cont_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+
+RSpec.describe "when I don't enter required information I get an error message" do
+  describe "and I am returned to the new review page" do
+    it "content is required to submit a review" do
+
+      shelter1 = Shelter.create(name: "Dog-Haven",
+                               address:  "1234 Barkers Way",
```

Comment:
Can we add a similar before :each structure we used in #create for uniformity during a refactor?